Controller/CkeditorAdminController.php

```diff
@@ -29,16 +30,21 @@ public function browserAction()
     {
         $this->checkIfMediaBundleIsLoaded();
 
-        if (false === $this->admin->isGranted('LIST')) {
-            throw new AccessDeniedException();
+        // NEXT_MAJOR: Remove this when dropping support for SF < 2.8 (change method signature)
```

Comment:
you mean inject the Request? if yes, we need to deprecate the usage of `get('request')` and 'get('request_stack')', didn't we?